crates/bevy_window/src/window.rs

```diff
@@ -124,6 +124,11 @@ impl Window {
         self.height
     }
 
+    #[inline]
+    pub fn resolution(&self) -> (u32, u32) {
```

Comment:
This is a personal taste thing, but I generally don't like returning tuples. This is a prime case for returning something like an `IVec2`, but `glam` (the math library we use) doesn't currently support integers.

I do eventually want to have integer vector types, either by expanding glam or by switching to something like `ultraviolet`. I think I'd rather wait for that before adding a `resolution()` method.

Reply:
My rationale for liking a tuple from `resolution()` is that it can be destructured. I do agree that a vector would be nicer though.

What about the inlines?

Reply:
Yup the inlines are good to go. Thanks!